src/suricata.c

```diff
@@ -524,7 +524,7 @@ static void SetBpfStringFromFile(char *filename)
     }
     bpf_len = st.st_size + 1;
 
-    bpf_filter = SCMalloc(bpf_len * sizeof(char));
```

Comment:
Suggest using `SCCalloc` here.

Reply:
I was just removing ` * sizeof(char)` 

Why should it be a SCCalloc ?